src/game_constants.h

```diff
@@ -36,6 +36,8 @@
 #define SEEX 12
 #define SEEY SEEX
 
+#define MAX_VIEW_DISTANCE ( SEEX * int( MAPSIZE / 2 ) )
```

Comment:
That would be what, 78?

Pretty sure you can only see 60 tiles. Maybe it should be called differently then? Max render distance!?